src/features/game/types/withdrawables.ts

```diff
@@ -646,6 +650,18 @@ const exoticCrops: Record<ExoticCropName, () => boolean> = {
   Chiogga: () => false,
 };
 
+const bait: Record<Bait, () => boolean> = {
+  Earthworm: () => false,
```

Comment:
The metadata says the bait are tradable but here they are marked as non-withdrawable.  Is the intent to unlock for withdraw at a future date? If so, maybe just code that in using the timestamp syntax now to [a] get the implementation done asap and [b] avoid constant discussions in discord about it. ;)